packages/devtools_app/assets/perfetto/dist/devtools_dark.css

```diff
@@ -0,0 +1,164 @@
+/**
```

Comment:
[Optional] this and other CSS files - alphabetize property names (I use the "CSS Alphabetize" VSCode plugin for this)

Reply:
nice! thanks